templates/email/default/archive.html

```diff
@@ -0,0 +1,58 @@
+[%
```

Comment:
Looking at the code `$cobrand->path_to_email_templates($lang_code)` seems to be returning `/templates/email/default` for some reason